mobx_codegen/lib/src/template/method_override.dart

```diff
@@ -10,10 +10,11 @@ class MethodOverrideTemplate {
 
   MethodOverrideTemplate.fromElement(ExecutableElement method) {
     // ignore: prefer_function_declarations_over_variables
-    final param = (ParameterElement elem) => ParamTemplate()
-      ..name = elem.name
-      ..type = findParameterTypeName(elem)
-      ..defaultValue = elem.defaultValueCode;
+    final param = (ParameterElement element) => ParamTemplate()
```

Comment:
Assuming this will work for the mixin based Store ? A test would be good to have

Reply:
Yep, it'll work everywhere. Added a @required annotation to a mixin action.

Technically, this is a breaking change. Users of the mixin style may have annotated params with @required, but because they weren't inherited until now, did not actually provide the args at the callsite.